src/transaction-flow/input/ProfileEditor/ProfileEditor-flow.tsx

```diff
@@ -286,7 +294,13 @@ const ProfileEditor = ({ data = {}, transactions = [], dispatch, onDismiss }: Pr
                     placeholder={placeholderForRecord(field)}
                     error={errorForRecordAtIndex(index)}
                     validated={isDirtyForRecordAtIndex(index)}
-                    onDelete={() => handleDeleteRecord(field, index)}
+                    onDelete={() => {
+                      if (isEthAddressRecord(field)) {
+                        updateRecordAtIndex(index, { ...field, value: '' })
+                      } else {
```

Comment:
This is unnecessary since the key is "descriptions" which means it will be a <textarea/> tag. Which will never be an address.